_projects/2022/100470958/100470958.Rmd

```diff
@@ -0,0 +1,389 @@
+  FIW.Status = c("F", "PF", "NF"),
+  order = 20,
+  GDP.var.NTL = c(175, 225, 275),
+  GDP.var.R = c(170, 220, 270)
+)
+
+countriestxt <- data.frame(
+  labels = c("Japan", "Britain", "United States", "China"),
+  FIW.Status = c("F", "F", "F", "NF"),
+  order = c(4, 11, 20, 34),
+  GDP.var.NTL = c(-6, 14, 25, 175),
+  GDP.var.R = c(-6, 14, 25, 175)
+  )
+
+c <- c +
+   geom_text(data = DF%>% filter(label == 1 & 
+          !(Country.Name %in% c("Japan", "Britain","United States", "China"))), 
+            aes(label = Country.Name), color = "#454545",
+            vjust = 1.5, hjust = -0.1, check_overlap = T) +
+  geom_text(data= countriestxt, aes(label = labels), color="#454545", hjust=0)+
+  geom_text(data = titles, aes(label = labels), fontface = "bold", size=4.5)
+
+c
+
+```
+To point out the averages, we draw a set of 3 dumbbells with a black border, and we include a label for them. Observe that we should create independent data frames, for annotations that require a specific location inside the chart. Then, we map these data frames with ggplot components.
+
+```{r out.width="100%"}
+
+av_data <- DF %>% 
+  group_by(FIW.Status)  %>%
+  summarise(
+    av.GDP.var.R = mean(GDP.var.R),
+    av.GDP.var.NTL = mean(GDP.var.NTL),
+    av.GDP.usd.R = mean(GDP.in.2015.USD.NTL)/1000000000000,
+    av.GDP.usd.NTL = mean(GDP.in.2015.USD.NTL)/1000000000000
+  )
+
+av_data <- av_data[c(1,3,2),]
+
+average <- data.frame(
+  Country.Name = c("Average", "Average", "Average"),
+  FIW.Status = c("F", "PF", "NF"),
+  GDP.in.2015.USD.NTL = as.vector(av_data$av.GDP.usd.NTL),
+  GDP.in.2015.USD.R = as.vector(av_data$av.GDP.usd.R),
+  GDP.var.R = as.vector(av_data$av.GDP.var.R),
+  GDP.var.NTL = as.vector(av_data$av.GDP.var.NTL),
+  order= c(22,19,17),
+  colorT = c("grey", "grey", "grey"),
+  colorB = c("#FFC259", "#F6423C","#FFA9A6"),
+  fill = c("black", "black", "black")
+)
+
+avTitles <- data.frame(
+  labels = c("Average", "Average", "Average"),
+  FIW.Status = c("F", "PF", "NF"),
+  order = c(20, 16, 14),
+  GDP.var.NTL = c(85, 138, 165),
+  GDP.var.R = c(80, 133, 160)
+  )
+
+c <- c +
+  geom_segment(data = average, aes(x=order, xend=order, y=GDP.var.NTL, 
+                                   yend=GDP.var.R), color="black") +
+  geom_point(data = average,aes(x=order, y=GDP.var.NTL, 
+             size=GDP.in.2015.USD.NTL, fill=FIW.Status), 
+             shape=21, color="black") + 
+  geom_point(data = average, aes(x=order, y=GDP.var.R, size=GDP.in.2015.USD.R), 
+             shape=21, color="black", fill="gray") +
+  geom_text(data= avTitles, aes(label= labels), fontface = "bold", 
+            size= 3.5, color = "black") 
+
+c
+```
+The autocratic category summary and the caption are added. We highlight that the summary is incorporated just to one facet. It means that we should specify the level of the factor that we want to use. To conclude, we combine the inset with the "Patchwork" library, specifying the desired position.
+
+```{r out.width="100%"}
```

Comment:
Please add `preview=TRUE` to the chunk options so that this chart is highlighted in the listing page. Also, I would try here `layout="l-page"` to see it larger. It is very nice!